litellm/proxy/secret_managers/get_azure_ad_token_provider.py

```diff
@@ -0,0 +1,24 @@
+import os
+from typing import Callable
+
+from azure.identity import DefaultAzureCredential
```

Comment:
can we move the `azure.` imports to inside the function. This way it's only required, if this approach is used

Reply:
✅